src/ripple/app/main/Application.cpp

```diff
@@ -937,8 +940,15 @@ class ApplicationImp
             std::chrono::seconds {config_->getSize (siSweepInterval)});
     }
 
+    LedgerIndex getMaxLedger() override
```

Comment:
Fixed. It's not minAllowed, it's minAllowed - 1 which doesn't flow off the tongue nicely. So I made it maxDisallowedLedger